tests/kafkatest/services/streams.py

```diff
@@ -123,6 +123,7 @@ def start_cmd(self, node):
               "INCLUDE_TEST_JARS=true %(kafka_run_class)s %(streams_class_name)s " \
               " %(kafka)s %(state_dir)s %(user_test_args)s %(user_test_args1)s %(user_test_args2)s" \
               " %(user_test_args3)s & echo $! >&3 ) 1>> %(stdout)s 2>> %(stderr)s 3> %(pidfile)s" % args
+        print(cmd)
```

Comment:
Having this in the output reduced my debug cycle time a lot. I've stripped out all my other debugging output. What do you think about leaving this in?